recipyCommon/config.py

```diff
@@ -49,10 +52,40 @@ def get_editor():
         if os.environ.get('EDITOR'):
             editor = '$EDITOR'
         else:
-            editor = None
+            editor = find_editor()
     return editor
 
 
+def find_editor():
+    """ Attemps to find a valid text editor by trying different executables (according to the operating system) and
+    checking for existence. """
+    platform = sys.platform
+
+    # open the empty file by trying different text editors according to the operating system
+    if "linux" in platform:
+        editor = _try_editors(["/usr/bin/editor", "nano", "vi", "emacs"])
+    elif "darwin" in platform:  # macintosh
+        editor = _try_editors(["nano", "vi", "emacs", "open -t"])
+    elif "win" in platform:  # windows
+        editor = _try_editors(["edit", "notepad", "notepad.exe"])
+    else:
+        raise RuntimeError("Cannot launch text editor for operating system %s. "
+                           "Try setting the editor in the ~/.recipy/recipyrc file" % platform)
+    if editor is None:
+        raise RuntimeError("Try setting the editor in the ~/.recipy/recipyrc file")
```

Comment:
Could you just add something like 'Cannot launch text editor' to the second error message here too?